src/hooks/useWithdrawFiat.tsx

```diff
@@ -0,0 +1,53 @@
+import { Client, Currency, TransferFailedException } from "@notuslabs/wallet";
+
+import { useSession } from "@/context/auth";
+
+type GetBalanceType = {
```

Comment:
Same thing about the suffix "Type", I saw that you removed on the ones I said, so you should remove it here too

```suggestion
type GetBalanceOptions = {
```